src/wasm-binary.h

```diff
@@ -473,6 +473,9 @@ class WasmBinaryWriter : public Visitor<WasmBinaryWriter, void> {
   WasmBinaryWriter(Module* input, BufferWithRandomAccess& o, bool debug) : wasm(input), o(o), debug(debug) {
     prepare();
   }
+  WasmBinaryWriter(Module& input, BufferWithRandomAccess& o, bool debug) : wasm(&input), o(o), debug(debug) {
```

Comment:
Ok, this PR looks great now, except that I think this is unnecessary - WebIDL should be able to handle a simple pointer? In fact that's the default behavior, so just removing the `[Ref]` should make it work.